<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>704</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Reco Workbench</string>
  </property>
  <property name="windowIcon">
   <iconset resource="application.qrc">
    <normaloff>:/assets/icons/ball.ico</normaloff>:/assets/icons/ball.ico</iconset>
  </property>
  <widget class="QWidget" name="central_widget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="central_layout">
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="raw_feed_layout">
        <item>
         <layout class="QVBoxLayout" name="depth_group_layout">
          <item>
           <widget class="QPushButton" name="show_depth_feed_button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Show Depth Feed</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="rgb_group_layout">
          <item>
           <widget class="QPushButton" name="show_rgb_feed_button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Show RGB Feed</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="playback_ctrl_layout">
        <item>
         <widget class="QPushButton" name="pause_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Pause Stream</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="play_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Play / Capture Stream</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="reconstruction_layout">
        <item>
         <widget class="QVTKWidget" name="qvtk_widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="reco_control_layout">
          <item>
           <widget class="QLabel" name="reco_label">
            <property name="text">
             <string>Reconstruction</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="reco_proc_group">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Process</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QPushButton" name="reco_proc_start_btn">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="toolTip">
                <string>Beging reconstruction process and start buffering the result.</string>
               </property>
               <property name="text">
                <string>Start</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="reco_proc_stop_btn">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Stop</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="reco_percent_label">
               <property name="toolTip">
                <string>% frames processed</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="text">
                <string>0.00%</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="reco_playback_group">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Playback</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QPushButton" name="reco_play_btn">
               <property name="toolTip">
                <string>Play back the reconstruction result</string>
               </property>
               <property name="text">
                <string>Play</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="reco_pause_btn">
               <property name="toolTip">
                <string>Pause reconstruction result playback</string>
               </property>
               <property name="text">
                <string>Pause</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="reco_rewind_btn">
               <property name="toolTip">
                <string>Rewind reconstruction result to beginning</string>
               </property>
               <property name="text">
                <string>Rewind</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>295</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1104</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menu_open_rgb_stream">
     <property name="title">
      <string>&amp;Open RGB-D Stream</string>
     </property>
     <addaction name="action_open_kinect_devices"/>
     <addaction name="action_open_hal_log"/>
     <addaction name="action_open_image_folder"/>
    </widget>
    <addaction name="menu_open_rgb_stream"/>
    <addaction name="action_open_calibration_file"/>
    <addaction name="action_close_stream"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="action_open_kinect_devices">
   <property name="text">
    <string>&amp;Kinect Devices</string>
   </property>
  </action>
  <action name="action_open_hal_log">
   <property name="text">
    <string>&amp;Log File</string>
   </property>
  </action>
  <action name="action_open_image_folder">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Image Folder</string>
   </property>
  </action>
  <action name="action_close_stream">
   <property name="text">
    <string>&amp;Close RGB-D Stream</string>
   </property>
  </action>
  <action name="action_open_calibration_file">
   <property name="text">
    <string>Open &amp;Calibration File</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header location="global">QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections/>
</ui>
